boondoggle/cli.py

```diff
@@ -42,7 +42,9 @@ def cli(context, profile, region):
               help="Path to a cloudformation file.")
 @click.option('--outputs-from',
               help="Use another stack's outputs as inputs to this one.")
-def ensure(context, name, params, url, file, outputs_from):
+@click.option('--useprevious',
```

Comment:
I don't see a use-case for --no-preserve so I think we can just add is_flag=True and use the option as a flag that you can exclude or include.  That way you don't have to pass True into it.